docs/design-structure-support/technical-assistance.md

```diff
@@ -3,58 +3,58 @@ sidebar_position: 1
+
+A startup will need to consider how they license and share their work. **There are three main things that are important to know about -- repositories, forums, and licensing.** More details will be covered further in the guide.
 
 ### Repositories
-**Do they have an existing repository for collaboration?** 
+**Do they have an existing repository for collaboration?**
 
-Open source projects often rely on many different people to contribute to the project.  Open source projects often rely on many different people to contribute to the project. This collaboration can take many different forms, but often, an open source project will have a public repository, which is a directory or storage space where projects live that anyone can access, which is sometimes shortened to “repo.”1 Common platforms for software collaboration are GitHub, Bitbucket, Sourceforge, OpenForge in India, CSDN in China, and iHub in Kenya.
+Open source projects often rely on many different people to contribute to the project. This collaboration can take many different forms, but often, an open source project will have a public repository, which is a directory or storage space where projects live that anyone can access, which is sometimes shortened to “repo.” Common platforms for software collaboration are GitHub, Bitbucket, Sourceforge, GitLab, OpenForge in India, CSDN in China, and iHub in Kenya.
 
 > **Q: How do you make sure that contributions are relevant, helpful, and in working order?**
 
 > _A: There is usually an approvals process where a collaborator can submit a contribution and other project leaders can decide which contributions are added to the final product.  Learn more about governance models in the next section of the guide._
 
-Resource: Interested in trying out open source for yourself, you can try making your own GitHub repository by following the steps on [this page](https://docs.github.com/en/get-started/quickstart/create-a-repo). 
+Resource: Interested in trying out open source for yourself, you can try making your own GitHub repository by following the steps on [this page](https://docs.github.com/en/get-started/quickstart/create-a-repo).
 
 ### Open Source Forums
-Before deciding to go open source, startups should try joining an open source community and talking to members to see if they enjoy working in this space. Most open source communities are very welcoming to newcomers!  Most open source projects will also have forums for discussion on any topics relating to the project.  
+Before deciding to go open source, startups should spend some time interacting with an existing open source community to better understand what it's like to work openly. It's best to select an open-source product already being used by the started and joining their community. Most open source communities are very welcoming to newcomers!  Most open source projects will also have forums for discussion on any topics relating to the project.
```

Comment:
I don't know if joining a forum should be required before starting an open source project. Added a more to frame this around interacting with the open source community.